build.gradle

```diff
@@ -28,6 +35,12 @@ dependencies {
 	implementation group: 'javax.persistence', name: 'javax.persistence-api', version: '2.2'
 }
 
+dependencyManagement {
```

Comment:
what this block does?

Reply:
Manages dependency for spring-boot-cloud...will probably work without it also

Reply:
okay